### ISSUE TITLE
Tutor: [A]ppend (Lesson 1.5) Goes to next time

### ISSUE THREAD

**Issue by btalpers** [neovim/neovim#4533]
- Neovim version:NVIM 0.1.2
  Build type: Debug
- Operating system: Fedora 23
- Terminal emulator: Terminator (xterm-256color)
### Actual behaviour

I highlight the 'e' with my cursor:
-> There is also some text miss  
-> There is also some text missing her**e**.

I hit 'j' to move up one line (cursor is where bold _ is, however no char is in that position)
-> There is also some text miss**_**  
-> There is also some text missing here.

I hit 'A' to append to the current line:
-> There is also some text miss  
-> There is also some text missing here.**_**

Since the tutorial allows you to move your cursor beyond where characters exists, the Append moves the cursor to the end of the next line for reading. This is not a bug in the functionality (as I imagine this behavior with Append is expected), however it is misleading since virtualedit is not on by default, and the tutorial explicitly states "It does not matter on what character the cursor is in that line."
### Expected behaviour

Append to the end of the current line / Make a note of this behavior in the tutorial.
### Steps to reproduce

Open nvim
:Tutor
scroll to lesson 1.5
move cursor to end of one of the complete lines
move cursor up one line (cursor will be one character beyond the last)
A - to Append

**Comment by fmoralesc** [comment]
Can confirm. I'll try to update the tutor plugin to fix this, I figured out what the issue is.

**Comment by dragon788** [comment]
Was having the same issue with lines ~ 141 and 143 on the Tutor. Additional behavior mine exhibited was when I pressed 'a' to append (I've used vim before so wasn't reading to use 'A'), it pulled in ```{expect:There is some test missing from this line.}``` and I had to actually delete that text in order to get it to pass, same with the second line. I also noticed the same behavior described above when following the directions and using A.
